cmd/swapd/main.go

```diff
@@ -81,14 +73,14 @@ const (
 	flagDevXMRMaker      = "dev-xmrmaker"
 	flagDeploy           = "deploy"
 	flagForwarderAddress = "forwarder-address"
-	flagTransferBack     = "transfer-back"
+	flagNoTransferBack   = "no-transfer-back"
```

Comment:
Ideally, boolean flags and boolean fields in Go structs use false for their default value. When we first changed the default behavior on this, it seemed like too much work to update all the code. In this PR, we were adding new code (`SwapdConfig`), in which I used `NoTransferBack` (so the uninitialized value was the was common behavior) and I decided to make everything consistent.